app/views/forums/index.html.haml

```diff
@@ -7,7 +7,7 @@
 - @forums.each do |forum|
   %h2= forum
   %p
-    = pluralize(forum.posts.size, "post")
+    = localize_plural(forum.posts, Post)
```

Comment:
My first approach was not to use the model name but a lot of tests broke because of empty collections. I assumed that this could be the case in every scenario so that is why I ended up doing this.